tools/mo/openvino/tools/mo/convert_impl.py

```diff
@@ -938,38 +1007,21 @@ def _convert(cli_parser: argparse.ArgumentParser, framework, args):
         inp_model_is_object = input_model_is_object(args)
         if inp_model_is_object:
             model_framework = check_model_object(args)
-            if model_framework == "pytorch" and not os.environ.get('USE_PYTORCH_FRONTEND'):
-
-                opset_version = None
-                if 'onnx_opset_version' in args and args['onnx_opset_version'] is not None:
-                    opset_version = args['onnx_opset_version']
-
+            print(args)
+            if model_framework == "pytorch":
+                use_pt_fe = os.environ.get('USE_PYTORCH_FRONTEND') or not args.get("use_legacy_frontend", False)
```

Comment:
Please do not use `use_legacy_frontend` option. It is only used for switching from ONNX FE to the legacy (Python) FE.

Reply:
@slyalin suggested to use `use_legacy_frontend`/`use_new_frontend `option to switch between PyTorch FE and converting via tmp ONNX model. So it looks correct to use `use_legacy_frontend` here. 

Check @slyalin's comments to this PR https://github.com/openvinotoolkit/openvino/pull/14380

Reply:
as agreed, we should use `use_legacy_frontend` for the switch in development purposes without mention in docs. This option will be removed later according to CVS-103726 (if the user wants to use onnx export, it should be done explicitly with all necessary parameters, which can not be guaranteed with pt->onnx->it way in conver_model due to limited parameters support).

Added TO DO comment in the code